node/cmd/guardiand/adminclient.go

```diff
@@ -168,6 +179,29 @@ func getPublicRPCServiceClient(ctx context.Context, addr string) (*grpc.ClientCo
 	return conn, c, err
 }
 
+func runSignWormchainValidatorAddress(cmd *cobra.Command, args []string) {
+	guardianKeyPath := args[0]
+	wormchainAddress := args[1]
+	if !strings.HasPrefix(wormchainAddress, "wormhole") || strings.HasPrefix(wormchainAddress, "wormholeval") {
+		log.Fatalf("must provide a bech32 address that has 'wormhole' prefix")
```

Comment:
`cobra.Command` also has a `RunE` field if we want to return errors here instead of doing `log.Fatalf`.